app/views/templates/provider_table_template.scala.html

```diff
@@ -21,28 +21,20 @@
 <div id="provider-results" role="region" aria-label="Search results">
 
     @if(showCount){
-        @result_count_template(providers.length, totalProviders, true)
+        @result_count_template(providers.length, totalProviders, doSearch = true)
     } else {
-        @result_count_template(providers.length, totalProviders, false)
+        @result_count_template(providers.length, totalProviders, doSearch = false)
     }
 
     <ul id="provider-table" class="govuk-list">
 
             @for((provider, index) <- providers.zipWithIndex){
                     <li class=" govuk-!-margin-left-0 govuk-table__cell">
                         @if(providerDetailsFilterFeature){
-                            <noscript>
                                 <a href="/making-tax-digital-software/software-information?providerName=@{provider.encodedName}"
                                    class="providers govuk-link govuk-!-font-weight-bold"
                                    data-journey-click="link:click:@{provider.name}">@provider.name
                                 </a>
-                            </noscript>
-                            <details class="govuk-details govuk-!-margin-0" data-module="govuk-details" role="group">
-                                <summary class="govuk-details__summary">
-                                    <span class="govuk-details__summary-text">@{provider.name}</span>
-                                </summary>
-                                <div class="govuk-details__text"></div>
-                            </details>
```

Comment:
Has that been confirmed by design? We could implement a drop down like that without javascript and save the user having to go to another page to see the information. Especially because the way the back link works currently will mean they lose all of their filters

Reply:
i just asked scott and he said that we should keep it the same as the non javascript version and get rid of the accordion since thats what design wanted.